client/less/toastr.less

```diff
@@ -8,3 +8,9 @@
 .notification-bar-message {
   padding-right: 2rem;
 }
+
+.notification-bar-action > a {
+  &:hover {
+    color: #007c00;
```

Comment:
Previous CSS made the link disappear against the `Toast` background when hovering